gen/installer/bash.py

```diff
@@ -336,19 +340,22 @@
     print_status $RC
     (( OVERALL_RC += $RC ))
 
-    for service in \
-      "80 mesos-ui" \
-      "53 mesos-dns" \
-      "15055 dcos-history" \
-      "5050 mesos-master" \
-      "2181 zookeeper" \
-      "8080 marathon" \
-      "3888 zookeeper" \
-      "8181 exhibitor" \
-      "8123 mesos-dns"
-    do
-      check_service $service
-    done
+    # Run service check on master node only
+    if [[ $AGENT_ONLY -eq 0 ]]; then
+        for service in \
```

Comment:
We need to still do a service check on agents. Agents require a different set of ports, which we should gather. Not having them should result in a failure of the preflight.